UPGRADING.md

```diff
@@ -0,0 +1,174 @@
+# Upgrading
+
+## 2.0.0
+
+the `cookbook` and `source` properites on resources allow you to override the template file the resource will use with your own
+
+### keepalived_global_defs
+
+- Removed property `config_name`, property `config_file` now will be the full name
+- Removed property `content`, this is now build up from the supplied properties
+- Removed property `exists`
+- Removed property `path`
+- Removed property `enable_snmp_keepalived`, no longer in keepalived man page
+- Added property `conf_directory`, defaulted to: `/etc/keepalived/conf.d`
```

Comment:
overall consistency, we have `conf_` vs ` config_`, we should pick one throughout I'd say `config_